static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-liveblog.js

```diff
@@ -51,24 +61,27 @@ const setupViewTracking = (el, test) => {
     });
 };
 
-const addEpicToBlocks = (epicHtml, test) =>
-    fastdom.write(() => {
-        const $blocksToInsertEpicAfter = $(`.${INSERT_EPIC_AFTER_CLASS}`);
-
-        $blocksToInsertEpicAfter.each(el => {
-            const $epic = $.create(epicHtml);
-
-            $epic.insertAfter(el);
-            $(el).removeClass(INSERT_EPIC_AFTER_CLASS);
-
-            setEpicLiveblogEntryTimeData(
-                $epic[0],
-                getLiveblogEntryTimeData(el)
-            );
-
-            setupViewTracking(el, test);
+const addEpicToBlocks = (
+    epicHtml: string,
+    test: ContributionsABTest
+): Promise<void> =>
+    fastdom
```

Comment:
@gustavpursche This seems a bit ugly, let me know if you have any good suggestions to simplify/clarify